app/models.py

```diff
@@ -26,31 +30,36 @@ def delete(self):
     @classmethod
     def get_by_id(cls, _id):
         return cls.query.filter_by(id=_id).first()
-    
+
     @classmethod
     def get_all(cls):
         return cls.query.all()
 
+
 class User(UserMixin, BaseModel):
+    __schema__ = UserSchema()
```

Comment:
This is wrong in the issue. The `__schema__` attribute needs to have the Class not the object as value